src/components/Tweet/Video.js

```diff
@@ -14,14 +14,14 @@ class Video extends React.Component {
     })
 
     let VideoComponent = (
-      <video src={videoSrc} controls={!gif} autoPlay={gif} loop={gif} style={styles.video}>
+      <video src={videoSrc} controls={!gif} autoPlay={this.props.autoPlay} loop={gif} style={styles.video}>
```

Comment:
can we turn this check into `autoPlay={gif || autoPlay}` so that gifs ALWAYS autoplay, and then videos will use the autoplay setting. you can also add `autoPlay` to the destructuring of `this.props` on line 8